images/linux/scripts/installers/oras-cli.sh

```diff
@@ -7,7 +7,7 @@
 source $HELPER_SCRIPTS/install.sh
 
 # Determine latest ORAS CLI version
```

Comment:
@suyash691 could you change the comment, please? We need to provide a comment why we install 0.12 with a link to the issue in the oras repo

Reply:
Done!